Authorization/Authorization/en.lproj/Localizable.strings

```diff
@@ -16,22 +16,35 @@
 "SIGN_IN.LOG_IN_BTN" = "Sign in";
 
 "ERROR.INVALID_EMAIL_ADDRESS" = "Invalid email address";
+"ERROR.INVALID_PASSWORD_LENGHT" = "Invalid password lenght";
+"ERROR.AUTH_PROVIDER" = "This %@ account is not linked with any %@ account. Please register.";
 "ERROR.INVALID_EMAIL_ADDRESS_OR_USERNAME" = "Invalid email or username";
-"ERROR.INVALID_PASSWORD_LENGTH" = "Invalid password length";
 
 "SIGN_UP.TITLE" = "Sign up";
 "SIGN_UP.SUBTITLE" = "Create new account.";
 "SIGN_UP.CREATE_ACCOUNT_BTN" = "Create account";
 "SIGN_UP.HIDE_FIELDS" = "Hide optional Fields";
 "SIGN_UP.SHOW_FIELDS" = "Show optional Fields";
+"SIGN_UP.SUCCESS_SIGNEDIN_LABEL" = "You've successfully signed in.";
```

Comment:
how about renaming it to `SIGN_UP.SUCCESS_SIGNIN_LABEL` and the same will go for sublabel.